doc/modules/compose.rst

```diff
@@ -52,9 +52,11 @@ The :class:`Pipeline` is built using a list of ``(key, value)`` pairs, where
 the ``key`` is a string containing the name you want to give this step and ``value``
 is an estimator object::
 
+    >>> from sklearn import set_config
     >>> from sklearn.pipeline import Pipeline
     >>> from sklearn.svm import SVC
     >>> from sklearn.decomposition import PCA
+    >>> set_config(display='diagram')
```

Comment:
Thanks for the PR @reshamas  but I don't think this is the right place for using the diagram rendering: this example is about a pipeline *creation*. Anything that's not directly related to that is noise and distracts from the original intent.

Reply:
@NicolasHug Where is a good place to put it?
Referring to:
>We probably need an example, or a user guide page, showing how different estimators are diagrammed, and perhaps also instructing users to use IPython.display's display function when needed.

Reply:
I answered in the original issue ;)

Reply:
That makes sense.  Keep the conversation with the original issue.